src/plugins/intel_cpu/src/nodes/mha.cpp

```diff
@@ -939,7 +939,7 @@ void MHA::prepareParams() {
 
     accPrecision0 = brg0Prc == Precision::I8 ? Precision::I32 : Precision::FP32;
 
-    size_t brg0BaseIdx = -1;
+    int64_t brg0BaseIdx = -1;
```

Comment:
```suggestion
    size_t brg0BaseIdx = std::numeric_limits<size_t>::max();
```
And the check on L966 ->:

```cpp
if (brg0BaseIdx == std::numeric_limits<size_t>::max())
```

Reply:
src/plugins/intel_cpu/src/emitters/x64/jit_snippets_emitters.cpp:792 also are updated.